server/server.go

```diff
@@ -224,9 +238,10 @@ func createTLSServer(address string, cert tls.Certificate) (*dns.Server, error)
 	return &dns.Server{
 		Addr: address,
 		Net:  "tcp-tls",
+		//nolint:gosec
 		TLSConfig: &tls.Config{
 			Certificates: []tls.Certificate{cert},
-			MinVersion:   tls.VersionTLS12,
+			MinVersion:   minTLSVersion(),
```

Comment:
Does the annotation work if you move to the end of line 244?  
`MinVersion:   minTLSVersion(), // nolint:gosec`

I'm asking cause that could make the linter complain for other issues in the `tls.Config` if any arise in the future.

Reply:
Not working 😅 
```
server/server.go:247:15: G402: TLS MinVersion too low. (gosec)
                TLSConfig: &tls.Config{
                        Certificates: []tls.Certificate{cert},
                        MinVersion:   minTLSVersion(), // nolint:gosec
                        CipherSuites: tlsCipherSuites(),
                },

server/server.go:249:35: directive `// nolint:gosec` is unused for linter "gosec" (nolintlint)
                        MinVersion:   minTLSVersion(), // nolint:gosec
                                                       ^
```